pages/api/oembed.ts

```diff
@@ -0,0 +1,29 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import probe from 'probe-image-size';
+import { HOST_URL } from '../../constants';
+
+export default async (
+  req: NextApiRequest,
+  res: NextApiResponse
+): Promise<void> => {
+  const { query } = req;
+
+  const [, playbackId] = (query.url as string).match(/(?:v\/)(.*)(\/?)$/) || [];
+
+  const imageUrl = `https://image.mux.com/${playbackId}/thumbnail.jpeg?width=480`;
+  const imageSize = await probe(imageUrl);
```

Comment:
can we re-purpose this probing functionality in here? it would get rid of a lot of ugly code:

https://github.com/muxinc/stream.new/blob/main/components/video-player.tsx#L69-L77